blocks/vehicle-selector/vehicle-selector.js

```diff
@@ -0,0 +1,241 @@
+/* eslint-disable max-len */
+import { createElement, stripEmptyTags } from '../../scripts/helpers.js';
+import { smoothScrollHorizontal } from '../../scripts/motion-helper.js';
+
+const blockName = 'vehicle-selector';
+
+function buildTabNavigation(tabItems, clickHandler) {
+  const tabNavigation = createElement('ul', { classes: `${blockName}__navigation` });
+  const navigationLine = createElement('li', { classes: `${blockName}__navigation-line` });
+
+  tabItems.forEach((tabItem, i) => {
+    const listItem = createElement('li', { classes: `${blockName}__navigation-item` });
+    const button = createElement('button');
+    button.classList.add('h5');
+    button.addEventListener('click', () => clickHandler(i));
+    // TODO make surer i target the right heading
```

Comment:
What does it mean? Can you implement it? Or if it is not needed remove the comment.